### DISCUSSION TITLE
[BUG]I had updated tp he latest chia version but since 1.2.8 version i am having sync problems

### DISCUSSION THREAD

**Discussion by yoga2005live** [Chia-Network/chia-blockchain#9143] [Support]
**Describe the bug**
My computer stop sync and after try many times it gets stuck
![error](https://user-images.githubusercontent.com/17745923/139603660-684c811b-b7e3-4595-bead-6283bccc37ed.PNG)

Just start and wait to sync it gest stuck 
![errora](https://user-images.githubusercontent.com/17745923/139603706-0e6eec4a-51e8-4ebb-817b-83775cf1bb65.PNG)

**Comment by loppefaaret** [comment]
do you have a low bandwidth internet connection ? if all you are seeing in the log is `wieght proof did not arrive in time` - that might be an issue ?
